src/components/me.js

```diff
@@ -33,7 +33,7 @@ module.exports = class Me extends User {
             this.removeChatFromSession(payload.chat);
         });
 
-        this.bindProtoPlugins();
+        this.onConstructed();
```

Comment:
I have only one question about this call. `Me` is extension of `User` and `User`  call same function in _constructor_. It will be called twice and generate two events for local user creation: `$.created.me` and `$.created.user`?

Reply:
Ah shit. Let me test.